tests/integration-tests/configs/common/common.yaml

```diff
@@ -602,11 +598,4 @@ resource_bucket:
       - regions: ["ap-southeast-1"]
         instances: {{ common.INSTANCES_DEFAULT_X86 }}
         oss: ["alinux2"]
-        schedulers: ["slurm", "awsbatch"]
-#nccl:
```

Comment:
This test was defined in the develop.yaml too and was testing alinu2 and ubuntu18